.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: Build Docker image
+
+on: [push]
+
+jobs:
+  build-and-testvariants:
+    name: Build image variants and run tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { variant: 'apache', test-files: ['apache-postgres'], docker-tag: 'roundcube-test-apache' }
+          - { variant: 'fpm', test-files: ['fpm-postgres'], docker-tag: 'roundcube-test-fpm' }
+          - { variant: 'fpm-alpine', test-files: ['fpm-postgres'], docker-tag: 'roundcube-test-fpm-alpine' }
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Get docker hub username
+        id: creds
+        run: echo '::set-output name=username::${{ secrets.DOCKER_PULL_USERNAME }}'
+      - name: Login to Docker Hub
+        if: steps.creds.outputs.username != ''
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_PULL_PASSWORD }}
+          password: ${{ secrets.DOCKER_PULL_USERNAME }}
```

Comment:
Looks like those two secrets have been swapped.

Reply:
Oh true, fixed it